src/css/support.js

```diff
@@ -0,0 +1,53 @@
+import jQuery from "../core.js";
+import document from "../var/document.js";
+import documentElement from "../var/documentElement.js";
+import support from "../var/support.js";
+
+( function() {
+
+var reliableTrDimensionsVal,
+	div = document.createElement( "div" );
+
+// Finish early in limited (non-browser) environments
+if ( !div.style ) {
+	return;
+}
+
+jQuery.extend( support, {
+
+	// Support: Firefox 70+
+	// Only Firefox includes border widths
+	// in computed dimensions. (gh-4529)
+	reliableTrDimensions: function() {
```

Comment:
Do we need the entirety of this test when we only test against Firefox behavior?

Reply:
Looking at the test I'm not sure what to remove. We need at least 3 elements, creating them with `createElement` prevents other issues as we've seen, the absolute styling is good practice to hide the table, and so on. Also, I removed the `isIE` check for this case so it's technically also covering IE11. I suppose I should note that in the Support comment in this module as well.

Reply:
Yeah, an IE comment would be good. If this test was possible to get shortened in a way that it would still cover non-IE browsers then we could check for IE using our `isIE` variable as we've done so far. But if the test cannot be made shorter, covering IE here is fine.